app/views/admin/articles/index.html.haml

```diff
@@ -8,9 +8,9 @@
       %thead.govuk-table__head
         %tr.govuk-table__row
           %th.govuk-table__header{scope: 'col', class: 'govuk-!-width-one-quarter'}
-            Description 
+            = link_to("Description", anchor: "published", :sort => "description")
```

Comment:
A nicer syntax for hashes where the keys are symbols is `key: value` rather than `:key => value` so I'd suggest all the `sort` calls in the `link_to` elements in this view be defined in this way. So:
```ruby
= link_to("Description", anchor: "published", sort: "description")
```